ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1103 @@
+             * used by its method name.
+             *
+             * Additionally, the URL to a graphic can be given on this form:
+             * `'url(graphic.png)'`. Note that for the image to be applied to
+             * exported charts, its URL needs to be accessible by the export
+             * server. Recommended size of image is 10x10px.
+             *
+             * @type      {string}
+             * @since     next
+             */
+            symbol: 'triangle-right',
+            /**
+             * A size of the separator
+             *
+             * @type      {number}
+             * @since     next
+             */
+            size: 10
+        },
+
+        /**
+         * Show full path or only a single button.
+         *
+         * @type      {boolean}
+         * @since     next
+         * @sample {highcharts} highcharts/breadcrumbs/show-full-path
+         *          Show full path.
+         */
+        showFullPath: false,
+
+        /**
+         * A CSS styles for all breadcrumbs.
+         *
+         * In styled mode, the breadcrumbs buttons are styled by the
+         * `.highcharts-range-selector-buttons .highcharts-button` rule with its
+         * different states.
+         *  @type {Highcharts.SVGAttributes}
+         *  @since     next
+         */
+        style: {
+            'fill': '#ffffff',
+            'position': 'absolute'
+        },
+
+        /**
+         * A useHTML property.
+         *
+         * @type      {boolean}
+         * @since     next
+         */
+        useHTML: false,
+
+        /**
+         * The zIndex of the group.
+         *
+         * @type      {boolean}
+         * @since     next
+         */
+        zIndex: 7
+    };
+
+    /* *
+     *
+     * Properties
+     *
+     * */
+    public chart: Chart = void 0 as any;
+    public breadcrumbsGroup: SVGElement = void 0 as any;
+    public breadcrumbsList: Array<Array<any>> = [];
+    public isDirty: boolean = true;
+    public isTreemap: boolean = void 0 as any;
+    public level: number = -1;
+    public options: Highcharts.BreadcrumbsOptions = void 0 as any;
+
+    public constructor(chart: Chart, userOptions?: Highcharts.BreadcrumbsOptions, isTreemap?: boolean) {
+        this.init(chart, userOptions as Highcharts.BreadcrumbsOptions, isTreemap as boolean);
+    }
+
+    public init(chart: Chart, userOptions: Highcharts.BreadcrumbsOptions, isTreemap: boolean): void {
+        const chartOptions = merge(Breadcrumbs.defaultBreadcrumbsOptions, userOptions);
+
+        this.chart = chart;
+        this.options = chartOptions || {};
+        this.isTreemap = isTreemap;
+    }
+
+    /**
+     * Align the breadcrumbs group depending on the alignment.
+     *
+     * @requires  modules/breadcrumbs
+     *
+     * @function Highcharts.Breadcrumbs#alingGroup
+     * @param {Highcharts.Breadcrumbs} this
+     *        Breadcrumbs class.
+     */
+    public alignGroup(this: Breadcrumbs): void {
```

Comment:
Most of this function is probably not needed. See how the current Drillown module uses the `relativeTo` option, which is passed directly on to `renderer.align`.